samples/ErrorPageSample/Startup.cs

```diff
@@ -1,12 +1,20 @@
 using System;
 using Microsoft.AspNet.Builder;
+using Microsoft.Framework.DependencyInjection;
 
 namespace ErrorPageSample
 {
     public class Startup
     {
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddEncoders();
+        }
+
         public void Configure(IApplicationBuilder app)
         {
+            app.UseRequestServices();
```

Comment:
Nope. We now have a default `IHtmlEncoder`. I have to update this PR with that change.